src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/caseworker/ChangeContactDetailsNotificationService.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.caseworker;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.ccd.domain.CCDCase;
+import uk.gov.hmcts.cmc.ccd.domain.CCDContactPartyType;
+import uk.gov.hmcts.cmc.claimstore.config.LoggerHandler;
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.EmailTemplates;
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.ContactDetailsChanged;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationService;
+import uk.gov.hmcts.cmc.claimstore.utils.CaseDetailsConverter;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIMANT_NAME;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.CLAIM_REFERENCE_NUMBER;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.DEFENDANT_NAME;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.EXTERNAL_ID;
+import static uk.gov.hmcts.cmc.claimstore.services.notifications.content.NotificationTemplateParameters.FRONTEND_BASE_URL;
+
+@Service
+public class ChangeContactDetailsNotificationService {
+
+    private static final String ERROR_MESSAGE =
+        "There was a technical problem. Nothing has been sent. You need to try again.";
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private final CaseDetailsConverter caseDetailsConverter;
+    private static final Logger logger = LoggerFactory.getLogger(LoggerHandler.class);
```

Comment:
This needs to change to 
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(ChangeContactDetailsNotificationService.class);
```

Reply:
done